db/structure.sql

```diff
@@ -3403,6 +3403,59 @@ COMMENT ON COLUMN public.reporting_facility_states.appts_scheduled_32_to_62_days
 
 
+--
+-- Name: reporting_facility_states_by_genders; Type: MATERIALIZED VIEW; Schema: public; Owner: -
+--
+
+CREATE MATERIALIZED VIEW public.reporting_facility_states_by_genders AS
+ WITH adjusted_outcomes AS (
+         SELECT reporting_patient_states.assigned_facility_region_id AS region_id,
+            reporting_patient_states.month_date,
+            count(DISTINCT reporting_patient_states.patient_id) FILTER (WHERE ((reporting_patient_states.htn_care_state = 'under_care'::text) AND ((reporting_patient_states.gender)::text = 'female'::text) AND (reporting_patient_states.htn_treatment_outcome_in_last_3_months = 'controlled'::text))) AS monthly_controlled_htn_female,
+            count(DISTINCT reporting_patient_states.patient_id) FILTER (WHERE ((reporting_patient_states.htn_care_state = 'under_care'::text) AND ((reporting_patient_states.gender)::text = 'male'::text) AND (reporting_patient_states.htn_treatment_outcome_in_last_3_months = 'controlled'::text))) AS monthly_controlled_htn_male,
+            count(DISTINCT reporting_patient_states.patient_id) FILTER (WHERE ((reporting_patient_states.htn_care_state = 'under_care'::text) AND ((reporting_patient_states.gender)::text = 'female'::text) AND (reporting_patient_states.htn_treatment_outcome_in_last_3_months = 'uncontrolled'::text))) AS monthly_uncontrolled_htn_female,
+            count(DISTINCT reporting_patient_states.patient_id) FILTER (WHERE ((reporting_patient_states.htn_care_state = 'under_care'::text) AND ((reporting_patient_states.gender)::text = 'male'::text) AND (reporting_patient_states.htn_treatment_outcome_in_last_3_months = 'uncontrolled'::text))) AS monthly_uncontrolled_htn_male
+           FROM public.reporting_patient_states
+          WHERE ((reporting_patient_states.hypertension = 'yes'::text) AND (reporting_patient_states.months_since_registration >= (3)::double precision))
+          GROUP BY reporting_patient_states.assigned_facility_region_id, reporting_patient_states.month_date
+        )
+ SELECT cal.month_date,
+    cal.month,
+    cal.quarter,
+    cal.year,
+    cal.month_string,
+    cal.quarter_string,
+    rf.facility_id,
+    rf.facility_name,
+    rf.facility_type,
+    rf.facility_size,
+    rf.facility_region_id,
+    rf.facility_region_name,
+    rf.facility_region_slug,
+    rf.block_region_id,
+    rf.block_name,
+    rf.block_slug,
+    rf.district_id,
+    rf.district_region_id,
+    rf.district_name,
+    rf.district_slug,
+    rf.state_region_id,
+    rf.state_name,
+    rf.state_slug,
+    rf.organization_id,
+    rf.organization_region_id,
+    rf.organization_name,
+    rf.organization_slug,
+    adjusted_outcomes.monthly_controlled_htn_female,
+    adjusted_outcomes.monthly_controlled_htn_male,
+    adjusted_outcomes.monthly_uncontrolled_htn_female,
+    adjusted_outcomes.monthly_uncontrolled_htn_male
```

Comment:
These field names match the conventions we have for `facility_state_groups` (which also do breakdowns by gender / diagnosis).